nemo/collections/asr/models/rnnt_models.py

```diff
@@ -130,6 +136,44 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self._optim_variational_noise_std = 0
             self._optim_variational_noise_start = 0
 
+    def _setup_loss(self, cfg: Optional[DictConfig]):
```

Comment:
name is a little misleading. How about extract_rnnt_cfg?

Reply:
Done